### PR TITLE
Support showNavButtons on categorisation stepper

### PR DESCRIPTION
**Pull request by chavalipraveen**
As per the discussion in the thread below:

https://spectrum.chat/jsonforms/general/materialui-categorization-stepper~5d14d1d3-be3e-4ace-8455-b2b5fbc5afbb

I used inline styles for the buttons as I couldn't use makeStyles inside the class component.

Please let me know if this looks okay.

Thanks!

### CODE REVIEW

**Comment by sdirix** [comment]
@chavalipraveen Thanks for the contribution :heart:. We'll definitely take a look soon. Would you be willing to also add unit tests for the new features in `MaterialCategorizationStepperLayout.test.tsx`?

File: packages/material/test/renderers/MaterialCategorizationStepperLayout.test.tsx [hunk 1]
```diff
@@ -406,4 +406,129 @@ describe('Material categorization stepper layout', () => {
...
+        <JsonFormsReduxContext>
+          <MaterialCategorizationStepperLayoutRenderer
+            {...layoutDefaultProps}
+            schema={fixture.schema}
+            uischema={uischema}
+          />
+        </JsonFormsReduxContext>
+      </Provider>
+    );
+    const beforeClick = wrapper.find(Stepper).props().activeStep;
```
Comment (sdirix) [inline packages/material/test/renderers/MaterialCategorizationStepperLayout.test.tsx:523]:
```suggestion
    const activeStepBeforeClick = wrapper.find(Stepper).props().activeStep;
```
I know that you just copied it from somewhere else in the test, but no reason to not improve on it ;)

**Comment by sdirix** [comment]
Note that we need a successful CLA check before we can merge this PR. It seems that one of your commits was not authored with the email address you registered with Github. I would like to suggest to either associate that email address with your account or changing the author of said commit (`7f9f292e9f21db4a0bdb5d47a660452bcb8199fb`).

**Comment by chavalipraveen** [comment]
@sdirix thank you for the review comments and suggestions. I have incorporated all of them. Please take a look when you get a chance.

Thanks!

**Pull request merged**